docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/amazon-quicksight.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Amazon QuickSight
+description: Amazon QuickSight overview
+template: concept-topic-template
+last_updated: Oct 24, 2024
+---
+
+Amazon QuickSight brings customizable and powerful business intelligence tools to the Back Office.
```

Comment:
as it will be the first page for the user, let's please add some more details on this feature for a better context and introduction:

Amazon QuickSight is a business analytics tool integrated directly into the Back Office. This feature lets users visualize data, create customizable dashboards, and perform deep analytics on sales, product performance, customer behaviors, and marketplace KPIs. By leveraging analytics, you can unlock new monetization opportunities and make more informed business decisions.

Highlights:
- Instant data connection with default dashboards, integrated directly within the Spryker platform for immediate insights and data export
- Go beyond simple visualization, enabling deep data analysis with preconnected Spryker DB
- High customizable, offering multiple options to tailor reports & analytics workflows 
- Extendable with multiple data sources so that you can aggregate data analysis across various systems